README.md

```diff
@@ -171,8 +171,30 @@ This builds an image for testing.
 This runs the image for testing.
 ```docker run test-image-v1```
 
-# Architectural Diagram
 
-- Link the images
+<div align="center">
+<h2 style=color:tomato><b>Architectural Diagram</b></h2>
+</div>
 
+<div align="center">
+<h3 style=color:SkyBlue> <b>Context Diagram</b> </h3>
+<img src="diagrams/context-diagram.png">
+<p> <strong> Figure: </strong> Context Diagram </p>
+</div>
 
+<div align="center">
+<h3 style=color:SkyBlue> <b>Container Diagram</b> </h3>
+<img src="diagrams/context-diagram.png">
+<p> <strong> Figure: </strong> Container Diagram </p>
+</div>
+
+<div align="center">
+<h3 style=color:SkyBlue> <b>Component Diagram</b> </h3>
+<img src="diagrams/context-diagram.png">
```

Comment:
Should update the image `src` for these two, they all point to the context diagram at the moment